packages/liquidator/package.json

```diff
@@ -31,7 +31,7 @@
   ],
   "bin": "index.js",
   "scripts": {
-    "test": "yarn mocha-test && yarn hardhat-test",
+    "test": "yarn mocha-test && mocha 'test/**/*.js'",
```

Comment:
If we're using mocha for _all_ the tests, can we combine them into a single call to mocha?

Reply:
good point! done.